app/subscriber/src/features/filter-media/FilterMedia.tsx

```diff
@@ -4,7 +4,7 @@ import { DateFilter } from 'components/date-filter';
 import { ContentListActionBar } from 'components/tool-bar';
 import { IContentSearchResult } from 'features/utils/interfaces';
 import React from 'react';
-import { useContent } from 'store/hooks';
+import { useContent, useSettings } from 'store/hooks';
```

Comment:
Change to useSettings. Added to settingsSlice the new field mediaTypesIdsAllSources to be saved in the state.